### PR TITLE
Cargo Crates Now Work!

### PR DESCRIPTION
**Pull request by Pandolphina**
Makes cargo crates work with access, adds guard gear crate, pistol crate and SMG crate. Removes wizard/witch crate. Cargo should work now, as soon as we get a shuttle.

### CODE REVIEW

File: code/datums/supplypacks/security.dm [hunk 1]
```diff
@@ -1,337 +1,259 @@
...
+    contains = list(/obj/item/weapon/storage/box/emps,
+                    /obj/item/weapon/grenade/smokebomb = 3,
+                    /obj/item/weapon/grenade/chem_grenade/incendiary)
+    cost = 20
+    containername = "\improper Special Ops crate"
+    hidden = 1
+
+/decl/hierarchy/supply_pack/security/pistol
+    name = "Pistol Crate"
+    contains = list(/obj/item/weapon/gun/projectile/pistol,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/security.dm]: Should instead be: /obj/item/weapon/gun/projectile/sec/sec

File: code/datums/supplypacks/security.dm [hunk 2]
```diff
@@ -1,337 +1,259 @@
...
+        /obj/item/clothing/suit/armor/pcarrier,
+        /obj/item/clothing/accessory/armorplate/medium,
+        /obj/item/clothing/accessory/armguards,
+        /obj/item/clothing/glasses/sunglasses/sechud/goggles,
+        /obj/item/clothing/accessory/legguards,
+        /obj/item/clothing/accessory/armor/tag/solgov/sec,
+        /obj/item/weapon/storage/belt/security/tactical,
+        /obj/item/clothing/accessory/storage/pouches,
+        /obj/item/weapon/gun/projectile/pistol,
+        /obj/item/ammo_magazine/mc9mm = 4,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/security.dm]: Should instead be /obj/item/ammo_magazine/c45m

File: code/datums/supplypacks/security.dm [hunk 3]
```diff
@@ -1,337 +1,259 @@
...
+        /obj/item/ammo_magazine/a10mm = 4)
+
+    cost = 40
+    containertype = /obj/structure/closet/crate/secure
+    containername = "\improper SMG Crate"
+    access = access_mtflvl3
+
+/decl/hierarchy/supply_pack/security/guard
+    name = "Guard Crate"
+    contains = list(/obj/item/clothing/under/solgov/utility/fleet/combat/security,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/security.dm]: /obj/item/clothing/under/scp/utility/security

File: code/datums/supplypacks/security.dm [hunk 4]
```diff
@@ -1,337 +1,259 @@
...
-					/obj/item/weapon/storage/photo_album,
-					/obj/item/device/reagent_scanner,
-					/obj/item/weapon/storage/briefcase/crimekit = 2)
-	cost = 50
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper Forensic equipment crate"
-	access = access_forensics_lockers
+    name = "Forensics - investigation equipment"
+    contains = list(/obj/item/weapon/storage/box/evidence = 2,
+                    /obj/item/weapon/cartridge/detective,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/security.dm]: PDA cartridge, can be removed.

File: code/datums/supplypacks/supply.dm [hunk 5]
```diff
@@ -1,87 +1,82 @@
...
-					/obj/item/clothing/glasses/meson)
-	cost = 15
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper Shaft miner equipment"
-	access = access_mining
+    name = "Miner equipment"
+    contains = list(/obj/item/weapon/storage/backpack/industrial,
+                    /obj/item/weapon/storage/backpack/satchel_eng,
+                    /obj/item/device/radio/headset/headset_cargo,
+                    /obj/item/clothing/under/rank/miner,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/supply.dm]: Can be removed.

File: code/datums/supplypacks/supply.dm [hunk 6]
```diff
@@ -1,87 +1,82 @@
...
-					/obj/item/clothing/glasses/material,
-					/obj/item/clothing/glasses/meson)
-	cost = 15
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper Shaft miner equipment"
-	access = access_mining
+    name = "Miner equipment"
+    contains = list(/obj/item/weapon/storage/backpack/industrial,
+                    /obj/item/weapon/storage/backpack/satchel_eng,
+                    /obj/item/device/radio/headset/headset_cargo,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/supply.dm]: Can be removed.

File: code/datums/supplypacks/supply.dm [hunk 7]
```diff
@@ -1,87 +1,82 @@
...
-					/obj/item/weapon/mining_scanner,
-					/obj/item/clothing/glasses/material,
-					/obj/item/clothing/glasses/meson)
-	cost = 15
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper Shaft miner equipment"
-	access = access_mining
+    name = "Miner equipment"
+    contains = list(/obj/item/weapon/storage/backpack/industrial,
+                    /obj/item/weapon/storage/backpack/satchel_eng,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/supply.dm]: Can be removed.

File: code/datums/supplypacks/supply.dm [hunk 8]
```diff
@@ -1,87 +1,82 @@
...
-					/obj/item/weapon/pickaxe,
-					/obj/item/weapon/mining_scanner,
-					/obj/item/clothing/glasses/material,
-					/obj/item/clothing/glasses/meson)
-	cost = 15
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper Shaft miner equipment"
-	access = access_mining
+    name = "Miner equipment"
+    contains = list(/obj/item/weapon/storage/backpack/industrial,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/supply.dm]: Can be removed.

File: code/datums/supplypacks/supply.dm [hunk 9]
```diff
@@ -1,87 +1,82 @@
...
+                    /obj/item/weapon/storage/backpack/satchel_eng,
+                    /obj/item/device/radio/headset/headset_cargo,
+                    /obj/item/clothing/under/rank/miner,
+                    /obj/item/clothing/gloves/thick,
+                    /obj/item/clothing/shoes/black,
```
Comment (JakeDaBoss) [inline code/datums/supplypacks/supply.dm]: Can be removed.

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]

**Comment by Pandolphina** [comment]
Now fixed. Bags/clothes removed, proper clothes/guns added.

**Comment by JakeDaBoss** [comment]
Does not compile, please check the compile logs.

**Comment by Pandolphina** [comment]
Should work, soon as you push your guns/clothes to GitHub.

**Comment by JakeDaBoss** [comment]
Please resolve conflicts, @Pandolphina

**Comment by Pandolphina** [comment]
The conflicting file is a CHANGELOG. Reee I'll change it in a few hours.

**Comment by Pandolphina** [comment]
Okay so for whatever reason my local git refuses to pull, push or do anything because of someone elses changelog so I'm just not gonna touch this. Please manually add the file/code thanks.

**Comment by JakeDaBoss** [comment]
I'm not going through 9 files. That's a monumental waste of my time. Close and re-open.

**Pull request closed without merging**